### ISSUE TITLE
How can I get my money back from MoonPay?

### ISSUE THREAD

**Issue by shirinscott5478** [tensorflow/serving#2770]
What is Moonpay customer service number?

Dial 🆑⭐1–850–468–9565 to get Moonpay customer service.

 

Can I contact Moonpay customer service live chat?

Presently, Moonpay does not have a live chat option on its website. You can, call 🆑⭐1–850–468–9565 and, write an email to https://support.moonpay.com/ or submit a request on their Support page.

 

Why am I unable to access my Moonpay account?

You will be unable to access your Moonpay account if you don’t have the recovery phase of your wallet. Note that there isn’t anything like 2-step verification or passwords in this platform. You can only use the 12-word recovery phase. So, if you have lost access to it, you cannot access the Moonpay account.

 

What is my Moonpay number?

On the Moonpay app, call 🆑⭐1–850–468–9565 choose the ‘Tokens’ tab from the main wallet screen. Choose your desired currency and hit the ‘Receive’ button below the wallet balance. You will see a QR code and, below it, an alphanumeric wallet here. Touch ‘Copy’ and store it somewhere safe.

 

How do I recover my money from Moonpay?

Call 🆑⭐1–850–468–9565 or On the Moonpay app, access the main screen. Now, choose either ‘Recover wallet’ or ‘Import wallet’ and click it. The account recovery process will get initiated. To fully recover your account, input the recovery or the backup phase. This is a series of 25, 24, or 12 words that will be given to you when you set up your account. Input this recovery phase in the correct order. Then enter a new password.

Summing up

Moonpay lets you store, buy, or move currency easily. But if you face any problems, reach out to them by submitting a ticket, dialing their support number, or writing an email. Bookmark this article and use it whenever you need customer support 🆑⭐1–850–468–9565.
